disorder-core/src/Disorder/Core/Property.hs

```diff
@@ -65,3 +74,9 @@ p1 .^. p2 = (p1 .||. p2) .&&. neg (p1 .&&. p2)
 infixr 1 <=>
 (<=>) :: (Testable p1, Testable p2) => p1 -> p2 -> Property
 a <=> b = (a .&&. b) .||. (neg a .&&. neg b)
+
+areEquivalent :: (Eq a, Show a) => [a] -> [a] -> Property
```

Comment:
Hm.. I don't mind any name as long as it is short :) In fact now I would rather go with something like `=\\=`... and if @olorin could rename `~~~` to `=~=` we would have a nice set of equals with different flavours.